src/controllers/processors/UpdateSessionFormSubmissionProcessor.ts

```diff
@@ -37,6 +38,9 @@ export abstract class UpdateSessionFormSubmissionProcessor<MODEL> implements For
                 appeal: {}
             } as ApplicationData);
 
+        loggerInstance()
```

Comment:
This log does not tell truth. What is being saved in session in not `applicationData` but `applicationData` extended with result of `this.prepareModelPriorSessionSave(applicationData.appeal, value)` call.

Reply:
Removed this in favour for log in controller that triggers this call